src/acl/external/eDirectory_userip/ext_edirectory_userip_acl.cc

```diff
@@ -1722,7 +1722,7 @@ MainSafe(int argc, char **argv)
                     local_printfx("BH message=\"(SearchFilterLDAP: %s)\"\n", ErrLDAP(x));
                 } else {
                     edui_ldap.err = -1;
-                    debug("SearchFilterLDAP(-, NULL) -> Length: %u\n", x);
+                    debug("SearchFilterLDAP(-, nullptr) -> Length: %u\n", x);
```

Comment:
This code change alters helper output. Please undo if this was not intentional. I am not requesting any changes here, and am not implying that this is the only functionality change in this PR.

Reply:
This was intentional. The code is nullptr not ```NULL/0``` nowdays, so fixing the debug message is useful.